### ISSUE TITLE
Missing contribution manual 

### ISSUE THREAD

**Issue by brybacki** [kubevirt/hostpath-provisioner-operator#224]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What happened**:
Bot shows this message for new PR's. The link points to non existing MD. 
```
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

memo Please follow instructions in the [contributing guide](https://github.com/kubevirt/hostpath-provisioner-operator/blob/master/CONTRIBUTING.md) to update your commits with the DCO

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).
```
**What you expected to happen**:

Correct link to CONTRIBUTING.MD, similar document example: https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#contributor-compliance-with-developer-certificate-of-origin-dco
. 

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Install tools:
- Others:
